Tests/Unit/NetworkServer/MessageProcessorJoinTest.cs

```diff
@@ -669,14 +656,15 @@ public async Task When_Join_With_Custom_Join_Update_Old_Desired_Properties()
 
         [Theory]
         // Base dr is 2
+        // In case the offset is invalid, we rollback to offset 0. for europe that means = to upstream
         [InlineData(0, DR2)]
         [InlineData(1, DR1)]
         [InlineData(2, DR0)]
         [InlineData(3, DR0)]
         [InlineData(4, DR0)]
-        [InlineData(6, DR0)]
-        [InlineData(12, DR0)]
-        [InlineData(-2, DR0)]
+        [InlineData(6, DR2)]
```

Comment:
Why?

Reply:
All thise offsets are invalid, that means we now threat as if there was no offseett. Previous logic seems to have defaulted to DR0?

Reply:
Actually, the expected values were never used on the left and the test was rather using txpk == rxpk. This is performing as expected